In [72]:
import pandas as pd
import numpy as np
import scipy
from scipy.stats import skew, kurtosis

In [73]:
cars_df = pd.read_csv('/Users/rahul/Downloads/Data Science Assignments/Basic_Statistics_Level_1/Q9_a.csv')
weight_df = pd.read_csv('/Users/rahul/Downloads/Data Science Assignments/Basic_Statistics_Level_1/Q9_b.csv')

In [74]:
cars_df, weight_df

(    Index  speed  dist
 0       1      4     2
 1       2      4    10
 2       3      7     4
 3       4      7    22
 4       5      8    16
 5       6      9    10
 6       7     10    18
 7       8     10    26
 8       9     10    34
 9      10     11    17
 10     11     11    28
 11     12     12    14
 12     13     12    20
 13     14     12    24
 14     15     12    28
 15     16     13    26
 16     17     13    34
 17     18     13    34
 18     19     13    46
 19     20     14    26
 20     21     14    36
 21     22     14    60
 22     23     14    80
 23     24     15    20
 24     25     15    26
 25     26     15    54
 26     27     16    32
 27     28     16    40
 28     29     17    32
 29     30     17    40
 30     31     17    50
 31     32     18    42
 32     33     18    56
 33     34     18    76
 34     35     18    84
 35     36     19    36
 36     37     19    46
 37     38     19    68
 38     39     20    32
 39     40     20    48
 40     41     2

In [75]:
cars_df = cars_df.set_index('Index')
weight_df = weight_df.rename(columns = {'Unnamed: 0': 'Index'})
weight_df = weight_df.set_index('Index')

In [76]:
cars_df, weight_df

(       speed  dist
 Index             
 1          4     2
 2          4    10
 3          7     4
 4          7    22
 5          8    16
 6          9    10
 7         10    18
 8         10    26
 9         10    34
 10        11    17
 11        11    28
 12        12    14
 13        12    20
 14        12    24
 15        12    28
 16        13    26
 17        13    34
 18        13    34
 19        13    46
 20        14    26
 21        14    36
 22        14    60
 23        14    80
 24        15    20
 25        15    26
 26        15    54
 27        16    32
 28        16    40
 29        17    32
 30        17    40
 31        17    50
 32        18    42
 33        18    56
 34        18    76
 35        18    84
 36        19    36
 37        19    46
 38        19    68
 39        20    32
 40        20    48
 41        20    52
 42        20    56
 43        20    64
 44        22    66
 45        23    54
 46        24    70
 47        24    92
 48        24    93


In [77]:
print(skew(cars_df, axis=0, bias=True))
print(kurtosis(cars_df, axis=0, bias=True))

[-0.11395477  0.78248352]
[-0.57714742  0.24801866]


Skew(Speed): -0.11395477
Skew(Distance): 0.78248352
Kurtosis(Speed): -0.57714742
Krtosis(Distance): 0.24801866

In [78]:
print(skew(weight_df, axis=0, bias=True))
print(kurtosis(weight_df, axis=0, bias=True))

[ 1.58145368 -0.60330993]
[2.72352149 0.81946588]


Skew(SP): 1.58145368
Skew(Weight): -0.60330993
Kurtosis(SP): 2.72352149
Krtosis(Weight): 0.81946588